src/components/landing/OfferingCard.js

```diff
@@ -0,0 +1,25 @@
+import { FaUserAlt } from "react-icons/fa";
+
+const OfferingCard = ({ title, description, icon }) => {
+  if (!icon && !title && !description) {
+    return <div></div>;
+  }
+  if (!title) {
+    title = "Title";
+  }
+  if (!icon) {
+    icon = <FaUserAlt className="size-12 text-[#4F9AD7]" />;
+  }
+  if (!description) {
+    description = "Description";
+  }
```

Comment:
If statements are unnecessary, as this is a static website, none of these props should be or ever will be blank